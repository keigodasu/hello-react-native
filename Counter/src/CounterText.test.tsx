import 'react-native';
import React from 'react';
import renderer from 'react-test-renderer';
import CounterText from '../src/CounterText';

describe('CounterText', () => {
  it('通常のCounterTextを描写できる', () => {
    const component = renderer.create(<CounterText count={0} />);
    expect(component).toMatchSnapshot();
  });
  it('カウントが10以上のときのテキストが表示されているCounterText', () => {
    const component = renderer.create(<CounterText count={10} />);
    expect(component).toMatchSnapshot();
  });
});
