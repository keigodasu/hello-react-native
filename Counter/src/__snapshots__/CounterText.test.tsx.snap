// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CounterText カウントが10以上のときのテキストが表示されているCounterText 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "justifyContent": "center",
      "width": 100,
    }
  }
>
  <Text>
    youre great
  </Text>
  <Text
    style={
      Object {
        "fontSize": 20,
      }
    }
  >
    10
  </Text>
</View>
`;

exports[`CounterText 通常のCounterTextを描写できる 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "justifyContent": "center",
      "width": 100,
    }
  }
>
  <Text
    style={
      Object {
        "fontSize": 20,
      }
    }
  >
    0
  </Text>
</View>
`;
